src/main/scala/com/databricks/labs/overwatch/validation/DeploymentValidation.scala

```diff
@@ -364,7 +364,12 @@ object DeploymentValidation extends SparkSessionWrapper {
    * @return
    */
   private def cloudSpecificValidation(config: MultiWorkspaceConfig): DeploymentValidationReport = {
-
+    if(config.auditlogprefix_source_path.getOrElse("").toLowerCase.equals("system")) {
```

Comment:
One more thing we need to validate is that we're only allowing customers to go from cloud->system tables, once the switch is made, we can't go back to cloud audit logs

Reply:
done